compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Error.hs

```diff
@@ -306,6 +313,12 @@ instance Pretty Error where
       "expected a type containing neither type variables nor quantifiers, but found: " <> pretty foundType
     EExpectedExceptionType foundType ->
       "expected an exception type, but found: " <> pretty foundType
+    EExpectedExceptionTypeHasNoParams modName exnName ->
+      "exception type should not have type parameters: " <> pretty modName <> "." <> pretty exnName
```

Comment:
"should" sounds like a friendly recommendation. Maybe "must" is more appropriate here?

Reply:
👍  changed!